src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+public abstract class Figure {
```

Comment:
<img width="795" alt="Screenshot 2022-03-28 at 14 19 37" src="https://user-images.githubusercontent.com/60258373/160387337-80db4a1b-9519-4483-8292-f68626d1507a.png">
Let's create two interfaces (one with a method for area calculation and the second with a method to draw a figure) then you have to override these methods in each heir of Figure